app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragmentViewModel.kt

```diff
@@ -32,9 +32,9 @@ class ProfileFragmentViewModel(private val authService: AuthService) : ViewModel
         compositeDisposable.add(authService.getProfile()
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
-                .doOnSubscribe({
+                .doOnSubscribe {
```

Comment:
WTC?

Reply:
> WTC?

Warning. Remove unnecessary parentheses as I mention in PR description. Should I recover it?

Reply:
Yes